app/src/main/AndroidManifest.xml

```diff
@@ -6,12 +6,10 @@
 
     <uses-sdk tools:overrideLibrary="android.support.test.uiautomator.v18" />
 
-    <uses-permission
-        android:name="android.permission.USE_FINGERPRINT"
```

Comment:
good catch on that. We clearly have no use for fingerprints... yet?

Reply:
we are good for now.

Reply:
👍